alchemist-full/PKGBUILD.template

```diff
@@ -35,8 +35,8 @@ package() {
     mkdir -p "${pkgdir}/usr/bin/"
 
     # Copy the application in usr/lib
-    cp -r "${srcdir}/usr/share/licenses/" "${pkgdir}/usr/share/alchemist/"
-    cp -r "${srcdir}/opt/${pkgname}/" "${pkgdir}/usr/lib/"
+    # cp -r "${srcdir}/usr/share/licenses/" "${pkgdir}/usr/share/alchemist/"
+    cp -r "${srcdir}/usr/lib/${pkgname}/" "${pkgdir}/usr/lib/"
```

Comment:
why?

Reply:
This change is related to the modification in the .rpm file, as indicated by this pull request. The PKGBUILD extracts the .rpm package (although .deb packages are also supported). RPM no longer utilizes /opt directory and lacks any licensing due to the issue previously mentioned, necessitating a template similar to what has been implemented with PKGBUILD.